city-of-la.md

```diff
@@ -0,0 +1,5 @@
+City of LA
+
```

Comment:
Hi! You may want to consider having subsections for different neighborhoods of the city.